### ISSUE TITLE
Adding IE specific versions of vendor CSS?

### ISSUE THREAD

**Issue by grantlucas** [ngbp/ngbp#113]
I'm currently working on a project using [Leaflet mapping library](http://leafletjs.com/) which has it's own needed CSS. I see I can add CSS in the vendor array but there are also some style sheets that are required for specific versions of IE.

Is there a way to add IE specific style sheets through the vendor array?

**Comment by joshdmiller** [comment]
@grantlucas Are you wanting something like an iterable array of IE-only scripts to put in the head of `index.html`?

**Comment by grantlucas** [comment]
Yeah, I think that's what I'm looking for. Some way to say, here's X stylesheets for IE8 and X for IE7 and then a way to loop through those to add them to `index.html`

For my specific case, I only need to include one IE stylesheet for IE8 and lower but obviously it'd be good to have it as generic as possible.

**Comment by joshdmiller** [comment]
@grantlucas IE is beyond my expertise as an open-source kind of guy, so I probably can't be too much help here. But I'd be happy to take a pull request. It _should_ be fairly straightforward; check out the logic [here](https://github.com/joshdmiller/ng-boilerplate/blob/v0.3.0-release/Gruntfile.js#L579) and the configuration [here](https://github.com/joshdmiller/ng-boilerplate/blob/v0.3.0-release/Gruntfile.js#L347) for inspiration.

**Comment by grantlucas** [comment]
Sounds good. Thanks for the resources to look at. Won't be for a little while but when I have a chance, I'll get a pull request together.
